README.md

```diff
@@ -1,92 +1,60 @@
-# HubSpot Visual Studio Code Extension
-For :rocket: fast HubSpot CMS Hub development.
+<h1 align="center">
+  <br>
+    <img src="https://github.com/HubSpot/hubspot-cms-vscode/blob/master/images/hubspot-logo.png?raw=true" alt="HubSpot Logo" width="150">
+  <br>
+  HubSpot for Visual Studio Code
+  <br>
+  <br>
+</h1>
 
-This extension currently supports:
-- Syntax highlighting for the [HubL templating language](https://designers.hubspot.com/docs/hubl/intro-to-hubl)
-- Autocomplete for common HubL tags, filters, and functions
-- Bracket wrapping, indentation, and other helpful editor features
+For fast HubSpot CMS Hub development right inside your editor.
 
-If you're new to CMS Hub, check out how to [get started with local development.](https://designers.hubspot.com/docs/tools/local-development)
+The new HubSpot panel in the activity bar provides developers access to features such as syntax formatters and highlighting for the [HubL templating language](https://designers.hubspot.com/docs/hubl/intro-to-hubl), as well as autocomplete for common HubL tags, filters, and functions. Additionally, it provides developers with quick views and commands to manage their connected accounts with HubSpot, extending some of the functionality of the HubSpot command line interface into the editor directly.
 
-## Getting Started
-This extension introduces new file languages: `HTML + HUBL` and `HTML + CSS`. In order to use these new features, you'll want to make sure you've told VSCode to use these languages. For one-off files, you can simply change the language in the lower right hand corner of your screen. However, we recommend adjusting your file associations under your User or Workspace preferences so HubL syntax highlighting will automatically be applied to all of your projects:
-- In VSCode, press `CMD` + `SHIFT` + `P` to open the command prompt
-- Search for and select the command `Preferences: Open Settings (UI)`
-- Choose either the "User" or "Workspace" tab
+<!-- TODO: Link 'HubSpot Developer Docs' text to CMS Dev Docs url when they are live -->
+To read more about this extension and its features, please see the documentation at the HubSpot Developer Docs. If you're new to CMS Hub, check out [how to get started with local development](https://designers.hubspot.com/docs/tools/local-development).
+
+## Settings
```

Comment:
I forgot if we talked about this or not. I wonder if we should just get rid of the beta option now that we can do beta releases? I'd still keep the ability to turn off hubl linting though. Does that sound ok?

Reply:
> I forgot if we talked about this or not. I wonder if we should just get rid of the beta option now that we can do beta releases? I'd still keep the ability to turn off hubl linting though. Does that sound ok?

This is a good point - probably could get rid of the Beta setting (I'll remove from the doc). And yes, keep HubL Linting. I'm thinking that we are going to probably have more settings to turn on/off features so that devs maybe have granular control over what the extension is doing for them (in the future). Like the ability to turn the HubL Prettier plugin off once that is in place, etc.

Reply:
> I think it would make sense to call out the "Beta" and "Hubl Linting" settings here

Based on Anthony's point -- I think I'm going to remove the language around beta, but I'm adding something in about settings to toggle